src/host_toolchains.py

```diff
@@ -26,8 +26,8 @@
 SETUP_TOOLCHAIN = os.path.join(CR_BUILD_DIR, 'toolchain', 'win',
                                'setup_toolchain.py')
 V8_SRC_DIR = os.path.join(WORK_DIR, 'v8', 'v8')
```

Comment:
The original author was trying to get away without having the cr-build-tools repo at all, and just using v8. But now that we depend on that anyway, I don't have any objection to this PR.